client/src/views/Simulation/components/VariablesPanel.vue

```diff
@@ -164,20 +159,40 @@
     @Action hideAllVariables;
     @Action showAllVariables;
 
+    calcPolygon (variable: HMI.SimulationVariable): void {
+      const numRuns = variable.values.length;
```

Comment:
You only use `numRuns - 1`, so why not just set it as `const numRuns = variable.values.length - 1;`?